lib/ssl/test/ssl_cert_tests.erl

```diff
@@ -516,3 +553,30 @@ group_config(Config, ServerOpts, ClientOpts) ->
                 {[{supported_groups, [x448, x25519]} | ServerOpts],
                  [{groups,"P-256:X25519"} | ClientOpts]}
         end.
+
+choose_sign_fun(#'RSAPrivateKey'{} = Key) ->
+    fun (_, {digest, Digest}, _HashAlgo, _KeyOpts, rsa) ->
+            public_key:encrypt_private(Digest, Key, [{rsa_pad, rsa_pkcs1_padding}]);
+        (_, Msg, HashAlgo, _KeyOpts, SignAlgo) ->
+            Options = signature_options(SignAlgo, HashAlgo),
+            public_key:sign(Msg, HashAlgo, Key, Options)
```

Comment:
To support old TLS-versions (pre TLS-1.2) we will also need
support for the encrypt/decrypt_private that only will need to
handle plain RSA keys (with param = NULL). This choice needs to be done in ssl.